src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -1111,7 +1115,6 @@ enum ValidationError {
     CANDIDATE_NAME_MISSING,
     CANDIDATE_NAME_INVALID,
     CANDIDATE_CODE_INVALID,
-    CANDIDATE_DUPLICATE_CODES,
```

Comment:
Note: there is no longer a distinction between duplicate names and duplicate codes. Names, codes, and aliases must all be unique.

Reply:
So what error does it throw if it hits a duplicate name then? CANDIDATE_NAME_INVALID? Might there still be a use for a CANDIDATE_DUPLICATE_NAMES error at least?

Reply:
`CANDIDATE_NAME_INVALID` was only ever doing a duplicate check, so I agree we should rename it as such.